vendor/github.com/circonus-labs/circonus-gometrics/submit.go

```diff
@@ -141,7 +141,8 @@ func (m *CirconusMetrics) trapCall(payload []byte) (int, error) {
 	client.CheckRetry = retryPolicy
 
 	attempts := -1
-	client.RequestLogHook = func(logger *log.Logger, req *http.Request, retryNumber int) {
+
+	client.RequestLogHook = func(logger retryablehttp.Logger, req *http.Request, retryNumber int) {
```

Comment:
the function interface for `RequestLogHook` changed in the updated go-retryablehttp